software/wmla120.py

```diff
@@ -1713,6 +1714,36 @@ def init_clients(self):
                 run = False
             print('All done')
 
+    def _gather_facts(self):
+        log = logger.getlogger()
+        run = True
```

Comment:
This is looking for gather_facts.yml in the power-up/software directory, but gather_facts.yml was committed in power-up/software/wmla120_ansible/ For test, I simply copied it to the software directory.  There doesn't seem to be anything in gather_facts.yml that is version specific so I would suggest just moving it to the software directory rather than modify this code.